parsec/core/mountpoint/fuse_operations.py

```diff
@@ -32,27 +32,27 @@ def is_banned(name):
 
 
 @contextmanager
-def translate_error():
+def translate_error(event_bus):
     try:
         yield
 
-    except FSError as exc:
+    except FSOperationLocalError as exc:
+        raise FuseOSError(exc.errno) from exc
```

Comment:
I think we should either check that `exc.errno` is not `None` or provide `FSOperationLocalError` with a default `ERRNO` (and `NTSTATUS`)

Reply:
55cd0f4430fd205d08ebecd47653113dfdf0bcdf